### ISSUE TITLE
Feedback module works as well.

### ISSUE THREAD

**Issue by VDBROEKU** [fdebrus/Nikobus-HA#14]
This is not really an issue 
I'm new to Home Assistant (and Linux). I had some issues installing the Nikobus HA repository. 
I'm using a RP4 using debian - OMV7 and also do not have a PC-link module, only a feedback module (05-207). 

I initially got an error "Could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0" 

I subsequently found some extra information in this link https://www.mforge.org/en/2017/05/19/control-relay-nikobus-with-home-of-apple/
and learned that I had to activate the serial port (which actually did not work with the information as mentioned in the above link) I had to use sudo apt install python3-serial. 
Subsequently, had an error "Module file not found: /config/nikobus_module_config.json" (and /config/nikobus_button_config.json) which was just me being stupid. (I forgot to remove .default in the files) 
The addresses of the different modules can be retrieved from the nikobus software. (rather straightforward but detailed in 
https://www.mforge.org/en/2017/05/19/data-extraction-from-your-nikobus-installation/)

Maybe you could update the installation instructions, maybe I'm just a noob. 

I will probably need some time to explore it more (you e.g. mention that I should be able to create virtual buttons, not that clear how, but I guess I will figure it out eventually) I'm not sure that your tool supports feedback leds either, but that's not important. 
To be honest, I never thought I could control my Nikobus installation of almost 20 years old via PC or mobile phone, but It works!
To summarize it somewhat disrespectfully, 'The Domotica of den Aldi' Rocks! 

Big thanks for the hard work. 

Kurt

**Comment by fdebrus** [comment]
Thanks for the feedback ! Glad you could make it work in your setup. Well done !

Side note, you can now expose Nikobus to homekit, google home, alexa, ... anything to make our old Nikobus a bit more modern. I personaly use homekit / siri with success. You can also link Nikobus buttons to anything outside Nikobus, eg I use a nikobus physical button to open/close my velux... 

I've another user using the feedback module. 
Do you have feedback led on the physical button ?
Do they get proper status "on/off" as you manipulate the output of the modules in homeassistant ?

Thanks
FreD

**Comment by VDBROEKU** [comment]
Hi Fred, 
As I said, I'm new still, trying to catch up. I do not use Google home or Siri. They just seemed a little redundant to me. I do have some Sonos gear which should be able to work with voice control, so why not. I also have some Velux windows, but those are not connected to home assistant either. It would indeed be nice if I could control them with some Nikobus buttons. 
Regarding the LEDs, I did not had the time to check. I only have feedback LEDs on the buttons in the top room of our house and I currently had a provisional setup using a COM-cable from the kitchen table to the nikobus feedback module on the electrical board next to the kitchen. I will need to make a more elegant solution first before I can experiment with it. 
I was just excited to get it to work.

**Comment by fdebrus** [comment]
If you have installed the latest version from my github, you have the nikobus “sniffer” included.

If you could do me a favor as I do not own a feedback module, nothing urgent, working on another matter for the time being, I will come back to Nikobus post vacation.

Instruct anyone in the home not to touch any button
Enable debug logging on the Nikobus integration

![image](https://github.com/fdebrus/Nikobus-HA/assets/33791533/52c70b24-3e5b-41e9-a2db-c6b67c9f81dd)

Push one button with feedback, ON briefly and OFF briefly, no long press.

Stop the debug logging

As you stop the debug logging, homeassistant will trigger a file download, send me that file and your 2 nikobus config files, AFTER the debugging session as your button will be discovered and created in the config file.

I'm only interested with the lines in the file that includes the keywork "nikobus", everything else you can delete for your privacy.

I would like to see how feedback module is managed. Those data will help.

Many Thanks.

**Comment by fdebrus** [comment]
Closing as inactive
